go/service/chat_local.go

```diff
@@ -1543,3 +1565,9 @@ func (h *chatLocalHandler) FindConversationsLocal(ctx context.Context,
 	res.RateLimits = utils.AggRateLimits(res.RateLimits)
 	return res, nil
 }
+
+// perfLog logs function call durations in ms.
+//    example usage: defer h.perfLog("MyFunc", time.Now())
+func (h *chatLocalHandler) perfLog(name string, start time.Time) {
+	h.G().Log.Debug("ChatPerf %s: %d", name, time.Since(start)/time.Millisecond)
```

Comment:
it has to be a parameter so that the defer works.

but yes, the internal ones inside NewConversationLocal don't use time.Now().